### ISSUE TITLE
Copula to address endogeneity

### ISSUE THREAD

**Issue by FloSchuberth** [M-E-Rademaker/cSEM#66]
We should implement the copula approach to address endogeneity, see Hult wt al. (2018)

**Comment by M-E-Steiner** [comment]
Yes, but probably something lower on the list of things that need to be done. I will move it to long-term projects.

**Comment by FloSchuberth** [comment]
Yes but You do not need to close it...

**Comment by FloSchuberth** [comment]
I reopen this issue as it seems to be quite straightforward to implement:
https://www.pls-sem.net/downloads/gaussian-copula-files/?fbclid=IwAR1SLpH0g76S6-1FjWA_ZPiYuUj7hUFE9epYiA0ZemC7NLiHY-ieFKU27w0

**Comment by FloSchuberth** [comment]
However, I am not sure whether it is a good idea to follow the original approach suggested by Hult et al. as they treat the construct scores as fixed instead of random (they use the scores in the bootstrap procedure instead of the original indicators).

**Comment by FloSchuberth** [comment]
We need to introduce a new expression in formula (similar to . for interactions) to indicate that for this variable(s) a copual should be calculated.
